src/entity/Settings.ts

```diff
@@ -52,6 +52,9 @@ export class Settings {
   @Column({ default: false })
   optOutWeeklyGoal: boolean;
 
+  @Column({ default: false })
+  optOutReadingStreak: boolean;
```

Comment:
Not blocking but did we decide anywhere that it is going to be a setting, maybe user `flags` is better place?

Reply:
Well I don't think there's been any decision on this, just since weekly goal was using settings went to use the same for this. 
U think it's best we use flags?

Reply:
Just a thought, but it is fine like this as well, especially since frontend is also done 👍

Reply:
Let's leave as is.